app/src/main/res/values/strings.xml

```diff
@@ -1178,8 +1178,8 @@
     <string name="migration_updating_app_button_text">Updating %s…</string>
     <!-- Text on the enabled button. Placeholder replaced with app name-->
     <string name="migration_update_app_button">Start %s</string>
-    <!-- Accessibility description text for completed migration item (e.g. Bookmarks, History). Placeholder example: complete -->
-    <string name="migration_icon_description">Migration status: %s</string>
+    <!-- Accessibility description text for a completed migration item -->
+    <string name="migration_icon_description">Migration completed</string>
```

Comment:
Localization won't be able to pick up changes to existing strings. You will need to add a new string name for it to be picked up. 

s/migration_icon_description/migration_icon_description_2